com.sleepfiles.OSCAR.yaml

```diff
@@ -0,0 +1,63 @@
+######################################################
+# Flatpak manifest for OSCAR
+# Erico Mendonca <erico.mendonca@gmail.com> Jan 2022 
+######################################################
+
+# Special OBS field because flatpak does not have a version field
+# Default will be '0' if the field is missing.
+#!BuildVersion: 1.3.1
+---
+app-id: com.sleepfiles.OSCAR
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: OSCAR
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=session-bus
```

Comment:
You need to use `--log-session-bus` to figure out which name you need to allow. Allowing `--session-bus` is mostly a no no.

Reply:
Done, limited to org.freedesktop.Notifications.